.gitignore

```diff
@@ -369,3 +369,5 @@ MigrationBackup/
 /docs/vendor
 /docs/.DS_Store
 /src/dymaptic.GeoBlazor.Core/Documentation/**/*
+
+package-lock.json
```

Comment:
Thanks for adding this!

Reply:
I believe that package lock is supposed to be committed to the repo. It keeps the resolved npm version numbers.

Reply:
ok, adding it back and removing it from .gitignore

Reply:
@morehavoc it's definitely a build-generated product, so it should be recreated with no issues if left out.

Reply:
@TimPurdum The `package-lock.json` file can be generated by the build, but it can also be consumed by the build. The `package-lock.json` contains the _exact_ versions of the packages that were used in the build, while the `package.json` file contains only the relative, hopefully semantically versioned package versions. 

The internet seems pretty torn on this (at least in some places). But the general consensus is that if you are building an NPM package you should *not* commit `package-lock.json` because it is ignored by the npm package manager when resolving dependencies for each library. But for building an app, which is what we have in terms of JS (it not a NPM module, it is an app) you should commit `package-lock.json`.

In my experience `package-lock.json` doesn't change unless you change something in `package.json` or something happened and a version of the package that was specified in package-lock doesn't exist on NPM any longer, which does happen sometimes.